assets/js/hooks/cell.js

```diff
@@ -88,6 +88,18 @@ const Cell = {
       `cells:${this.props.cellId}`,
       (event) => this.handleCellEvent(event)
     );
+
+    // Ensure the editor remains visible if the visual viewport changes.
+    // This occurs when a virtual keyboard opens.
+    window.visualViewport.addEventListener("resize", () => {
```

Comment:
We need to remove the handler on `destroyed()`. We don't need to do that for elements inside the hook, because they (and hence their handlers) are removed with the hook. Global handlers are not removed automatically, so keeping them would lead to memory leak. You can see `session.js` for a reference :)